libs/SalesforceAnalytics/src/com/salesforce/androidsdk/analytics/logger/FileLogger.java

```diff
@@ -102,11 +108,11 @@ public int getMaxSize() {
      *
      * @param size Maximum number of log lines.
      */
-    public synchronized void setMaxSize(int size) {
+    public void setMaxSize(int size) {
         if (size < 0) {
             size = 0;
         }
-        maxSize = size;
+        storeFileLoggerPrefs(size);
```

Comment:
Nope - it should be used this time. It's set at the end of `storeFileLoggerPrefs` [here](https://github.com/forcedotcom/SalesforceMobileSDK-Android/pull/1421/files#diff-a6cbed71575822a567cf352b98d0d547R269).